### PR TITLE
Add SurveyGroupSurveys and update Supply Guide

### PR DESCRIPTION
**Pull request by snchildress**
Added the SurveyGroupSurveys property to the /Surveys/SurveyGroups/ call and cleaned up the recommended implementation of Survey Groups in the Supply Guide.

### CODE REVIEW

File: source/includes/guides/_supply.md [hunk 1]
```diff
@@ -204,23 +204,23 @@ __Objective: Earn the most revenue with the least amount of clicks by reviewing
...
+In this phase we’ll explain how to handle Survey Groups, Recontact Studies, and Custom Qualifications.
 
 #### Survey Groups
 
-The [Show an Allocated Survey](#get-show-an-allocated-survey) call returns the property `SurveyGroup`.  This information can be used to avoid sending respondents to a survey that is included in a survey group that contains a survey which they have previously  attempted. Buyers typically remove older surveys from the survey group over time as the lockout period ends and the respondent can then attempt new surveys once again.
+Fulcrum buyers place surveys in survey groups to avoid duplication across multiple surveys. Buyers may add or remove surveys from survey groups as their deduplication needs change. Suppliers should avoid sending the same respondent to all other surveys in a survey group if that respondent previously attempted one of that group's surveys. The [Show an Allocated Survey](#get-show-an-allocated-survey) call returns the property `SurveyGroupExists`, which can be used to determine if the [List a Survey's Groups](#groups) call should be made for that survey.
```
Comment (brenankeller) [inline source/includes/guides/_supply.md]: @snchildress Is it intentional to block after a single entry although buyer config may only block if they have completed?

File: source/includes/guides/_supply.md [hunk 2]
```diff
@@ -204,23 +204,23 @@ __Objective: Earn the most revenue with the least amount of clicks by reviewing
...
 
 #### Survey Groups
 
-The [Show an Allocated Survey](#get-show-an-allocated-survey) call returns the property `SurveyGroup`.  This information can be used to avoid sending respondents to a survey that is included in a survey group that contains a survey which they have previously  attempted. Buyers typically remove older surveys from the survey group over time as the lockout period ends and the respondent can then attempt new surveys once again.
+Fulcrum buyers place surveys in survey groups to avoid duplication across multiple surveys. Buyers may add or remove surveys from survey groups as their deduplication needs change. Suppliers should avoid sending the same respondent to all other surveys in a survey group if that respondent previously attempted one of that group's surveys. The [Show an Allocated Survey](#get-show-an-allocated-survey) call returns the property `SurveyGroupExists`, which can be used to determine if the [List a Survey's Groups](#groups) call should be made for that survey.
 
-Below is recommended process to check and update survey groups every 5 minutes:
+Below is the recommended process to check and update survey groups every 10 minutes:
```
Comment (brenankeller) [inline source/includes/guides/_supply.md]: What's behind the decision to reduce frequency?

File: source/includes/guides/_supply.md [hunk 3]
```diff
@@ -204,23 +204,23 @@ __Objective: Earn the most revenue with the least amount of clicks by reviewing
...
-3. If `SurveyGroupExists` = `0` then no additional steps are needed as the survey is not in a survey group
+1. Make the the [List Exchange Surveys](#get-list-exchange-surveys) and the [Show an Allocated Survey](#get-show-an-allocated-survey) calls.
+2. Check the `SurveyGroupExists` property. `0` or `1` indicates whether there is a survey group(s) associated with the survey, where `0` represents `false` and `1` represents `true`.
+3. If `SurveyGroupExists` = `0` then no additional steps are needed.
 4. If `SurveyGroupExists` = `1` then the survey is in a survey group and you should make the [List a Survey’s Groups](#groups) call for that survey.
-5. Add the groups's survey number(s) to your survey group table and continue to check that survey number until it is removed from the survey group—the response will be `null`.
-6. Do not send the same respondent to any survey in that survey group until you get a `null` response for that survey.
+5. Store the `SurveyGroupID` to your survey groups table with every survey number returned in the `SurveyGroupSurveys` array. Remove or inactivate survey numbers not returned in that array for that `SurveyGroupID` from your survey groups table.
```
Comment (brenankeller) [inline source/includes/guides/_supply.md]: A "survey groups table" is pretty far into implementation details. Their database schema may not contain a survey groups table or they may not have tables at all in the case of graph databases. Is there a way to put this in more general terms?

File: source/includes/guides/_supply.md [hunk 4]
```diff
@@ -204,23 +204,23 @@ __Objective: Earn the most revenue with the least amount of clicks by reviewing
...
+6. Do not send a respondent to more than one survey in each survey group. If a survey is removed from a survey group, you may begin sending respondents you sent to that particular survey to other survey numbers remaining in that group.
 
 
-`SurveyGroup` and `SurveyGroupID` on the [List Exchange Surveys](#get-list-exchange-surveys) and the [Show an Allocated Survey](#get-show-an-allocated-survey) calls will now always return null as per June 25th changes. You should no longer use these fields.
+You should no longer use the `SurveyGroup` and `SurveyGroupID` properties returned on the [List Exchange Surveys](#get-list-exchange-surveys) and the [Show an Allocated Survey](#get-show-an-allocated-survey) calls. These fields will always return `null` per their removal on June 25, 2016.
```
Comment (brenankeller) [inline source/includes/guides/_supply.md]: Maybe change `removal` to `deprecation`?

File: source/includes/supply/_groups.md [hunk 5]
```diff
@@ -2,14 +2,15 @@
...
 
-The [Surveys/AllOfferwall](#get-list-exchange-surveys) and [SupplierAllocations/BySurveyNumber](#get-show-an-allocated-survey) calls return a SurveyGroupExists property which will indicate if the survey is in a survey group. The Surveys/SurveyGroups call can then be made to identify and track survey group details. You will need to continue to check the survey number and do not send the same respondent to any survey in that survey group until the original survey number sent to returns "null".
+The [Surveys/AllOfferwall](#get-list-exchange-surveys) and [SupplierAllocations/BySurveyNumber](#get-show-an-allocated-survey) calls return a SurveyGroupExists property which will indicate if the survey is in a survey group. The Surveys/SurveyGroups call can then be made to identify and track survey group details. You will need to continue to check the survey number and do not send the same respondent to any survey in that survey group until the original survey number sent to returns `"SurveyGroupExists": 0` from any of the [Supplier Allocations](#allocations) calls.
```
Comment (brenankeller) [inline source/includes/supply/_groups.md]: Careful with the slash here. I read it as a path.

File: source/includes/guides/_supply.md (hunk 1 above)
Comment (snchildress) [inline source/includes/guides/_supply.md]: Yes... until we expose re-entry configurations, it is preferable for both buyers and suppliers to never send in a respondent that may DQ for survey group terms. Sending in respondents that DQ for survey groups crushes the study by lowering conversion very drastically.

File: source/includes/guides/_supply.md (hunk 2 above)
Comment (snchildress) [inline source/includes/guides/_supply.md]: Consistency with another frequency recommendation in the docs.

File: source/includes/guides/_supply.md (hunk 3 above)
Comment (snchildress) [inline source/includes/guides/_supply.md]: Agreed - I just re-used verbiage that was already there. If you can think of something better, let's do it.

File: source/includes/supply/_groups.md (hunk 5 above)
Comment (snchildress) [inline source/includes/supply/_groups.md]: Haha I enjoy your criticisms of the old verbiage! Agreed though.

File: source/includes/guides/_supply.md (hunk 3 above)
Comment (brenankeller) [inline source/includes/guides/_supply.md]: > Store the `SurveyGroupID` and associated surveys from the `SurveyGroupSurveys` array. On future polls remove or inactivate survey numbers not returned for that `SurveyGroupID`.

File: source/includes/guides/_supply.md (hunk 2 above)
Comment (wkuckro) [inline source/includes/guides/_supply.md]: There is still a 30 min cache on removing a survey from a survey group in this call. 

Example:
pasted image at 2017_01_24 17_34

the survey listed was in both survey groups. I deleted it from SurveyGroupID 31112 via the UI, and then reran the call and now it is in neither, but still gives me these results until I wait 30 min

**Pull request merged**